src/FakeItEasy/Configuration/BuildableCallRule.cs

```diff
@@ -77,7 +77,7 @@ public void UseApplicator(Action<IInterceptedFakeObjectCall> newApplicator)
         {
             if (this.wasApplicatorSet)
             {
-                throw new InvalidOperationException("The behavior for this call has already been defined");
+                throw new InvalidOperationException(ExceptionMessages.CallBehaviorAlreadyDefined);

```

Comment:
General comment on this commit: technically FxCop just complained about string literals in the constructors. I hunted down other constructed arguments and moved them to `ExceptionMessages` as well.